x-pack/plugins/infra/public/pages/logs/toolbar.tsx

```diff
@@ -53,22 +61,26 @@ export const LogsToolbar: React.SFC = () => (
         <LogCustomizationMenu>
           <WithLogMinimap>
             {({ availableIntervalSizes, intervalSize, setIntervalSize }) => (
-              <LogMinimapScaleControls
-                availableIntervalSizes={availableIntervalSizes}
-                setIntervalSize={setIntervalSize}
-                intervalSize={intervalSize}
-              />
+              <I18nProvider>
+                <LogMinimapScaleControls
+                  availableIntervalSizes={availableIntervalSizes}
+                  setIntervalSize={setIntervalSize}
+                  intervalSize={intervalSize}
+                />
+              </I18nProvider>
             )}
           </WithLogMinimap>
           <WithLogTextview>
             {({ availableTextScales, textScale, setTextScale, setTextWrap, wrap }) => (
               <>
-                <LogTextWrapControls wrap={wrap} setTextWrap={setTextWrap} />
-                <LogTextScaleControls
-                  availableTextScales={availableTextScales}
-                  textScale={textScale}
-                  setTextScale={setTextScale}
-                />
+                <I18nProvider>
```

Comment:
do not use I18nProvider here. Find ReactDOM.render call and use I18nProvider there